Earthfile

```diff
@@ -325,11 +325,13 @@ install-docs-deps:
     COPY docs/package.json ./docs/package.json
     COPY docs/yarn.lock ./docs/yarn.lock
     RUN cd docs && yarn install
+    COPY target/doc/dbsp ./docs/rust
 
 build-docs:
     FROM +install-docs-deps
     COPY docs/ docs/
     COPY ( +build-manager/dbsp_pipeline_manager ) ./docs/dbsp_pipeline_manager
+    RUN cargo doc
```

Comment:
It's not a good idea to call doc here since it inherits from     FROM +install-docs-deps which has not built target/ yet. ideally `cargo doc` can be called inside the build-dbsp target and saved using SAVE ARTEFACT and then copied here using something like `COPY ( +build-dbsp/docs ) ...`